packages/manager-react-components/src/components/datagrid/datagrid.component.tsx

```diff
@@ -160,6 +164,7 @@ export const Datagrid = <T,>({
   search,
   topbar,
   totalItems,
+  isFetchAll,
```

Comment:
name this `isVirtualized` instead, it doesn't have to be tied to the fact that all the data is all loaded or not